.teamcity/_self/projects/WPComPlugins.kt

```diff
@@ -1,17 +1,25 @@
-package _self.builds
+package _self.projects
 
 import _self.bashNodeScript
-import jetbrains.buildServer.configs.kotlin.v2019_2.AbsoluteId
-import jetbrains.buildServer.configs.kotlin.v2019_2.BuildStep
 import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
+import jetbrains.buildServer.configs.kotlin.v2019_2.Project
 import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.PullRequests
 import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.commitStatusPublisher
 import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.pullRequests
 import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.vcs
 
-object WPComPlugins_EditorToolKit : BuildType({
-	name = "Editor ToolKit"
+object WPComPlugins : Project({
```

Comment:
This is a copy/paste from the same project in settings.kts, plus the addition of id("WPComPlugins")